src/shared/utils/upgradeForm.ts

```diff
@@ -142,16 +144,16 @@ export function shouldRenderCancelLink({
   trialStatus,
 }: {
   cancelAtPeriodEnd: boolean
-  plan: Plan
+  plan: PlanName
```

Comment:
Keeping this as Plan would help a lot in the future as well since we'll be using properties off the plan instead of the planName